.env.local

```diff
@@ -5,4 +5,3 @@ APP_URL=http://localhost
 
 # required for Laravel defautlst to work
 DB_CONNECTION=sqlite
-DB_DATABASE=database/database.sqlite
```

Comment:
@TomasVotruba laravel require absolute path when defined now, so use default from framework which an absolute path to database/database.sqlite.